openedx/core/djangoapps/user_api/preferences/tests/test_api.py

```diff
@@ -191,16 +219,18 @@ def test_update_user_preferences_errors(self, user_preference_save, user_prefere
             self.test_preference_key: "new_value"
         }
         with self.assertRaises(UserNotFound):
-            update_user_preferences(self.user, update_data, username="no_such_user")
+            update_user_preferences(self.user, update_data, user="no_such_user")
+        with self.assertRaises(UserNotFound):
+            update_user_preferences(self.user, update_data, user="no_such_user")
```

Comment:
Aren't L223-224 exactly the same as the lines above?  Typo?  Or are we intentionally calling it twice?